server/model/News.js

```diff
@@ -14,6 +14,10 @@ module.exports = (sequelize, DataTypes) => {
       type: DataTypes.STRING(5000),
       allowNull: false,
     },
+    icon: {
+      type: DataTypes.STRING,
+      allowNull: false,
```

Comment:
Hier könnte man evtl. noch einen Default spendieren

Reply:
Hatte ich auch überlegt, aber dann ist in der db fest verankert was das frontend können muss. Vielleicht wechselt ja mal das icon set.